commit a742cb3dd6d2e30bc3792d0d1741ee5fca887b1d
Author: Balló György <ballogyor@gmail.com>
Date:   Tue Oct 7 22:18:22 2014 +0200

    lxsession-edit: Fix segfault on exit when kf is not set.

diff --git a/lxsession-edit/lxsession-edit.c b/lxsession-edit/lxsession-edit.c
index 7e2a4c1..88fe9d1 100644
--- a/lxsession-edit/lxsession-edit.c
+++ b/lxsession-edit/lxsession-edit.c
@@ -82,11 +82,12 @@ int main(int argc, char** argv)
     load_autostart(session_name);
     gtk_tree_view_set_model( (GtkTreeView*)autostarts, (GtkTreeModel*)get_autostart_list() );
 
+    kf = g_key_file_new();
+
     /* if we are running under LXSession */
     if( g_getenv("_LXSESSION_PID") )
     {
         /* wm settings (only show this when we are under lxsession) */
-        kf = g_key_file_new();
         cfg = g_build_filename( g_get_user_config_dir(), "lxsession", session_name, CONFIG_FILE_NAME, NULL );
         loaded = g_key_file_load_from_file(kf, cfg, 0, NULL);
         if( !loaded )
